src/aap_eda/settings/defaults.py

```diff
@@ -138,7 +138,7 @@
 # ---------------------------------------------------------
 DEPLOYMENT_TYPE: str = "podman"
 WEBSOCKET_BASE_URL: str = "ws://localhost:8000"
-WEBSOCKET_SSL_VERIFY: str = "yes"
+WEBSOCKET_SSL_VERIFY: bool = True
```

Comment:
I always understood that it should be `Union[str|bool]`
Where it is converted to string? because in container request and what it is passed to ansible-rulebook it is a string.

Reply:
When we call ansible-rulebook cli, all boolean becomes str. See this log:
```
Container args ['ansible-rulebook', '--worker', '--websocket-ssl-verify', 'False', '--websocket-url'
```

Reply:
I tested various combinations (including "yes" in yaml) and proved all work no matter what's the input. But changing to Union[bool, str] does not hurt.

Reply:
Before we called function `_get_boolean` for this kind of situation, right? it works for both boolean and str types.
```
def _get_boolean(name: str, default=False) -> bool:
    value = settings.get(name, default)
    if isinstance(value, str):
        value = utils.str_to_bool(value)
    if not isinstance(value, bool):
        raise ImproperlyConfigured("{name} setting must be a boolean value.")
    return value


DEBUG = _get_boolean("DEBUG")
```

Reply:
thanks!